README.md

```diff
@@ -94,6 +103,18 @@ export GITHUB_AUTH_TOKEN=<your access token>
 set GITHUB_AUTH_TOKEN=<your access token>
 ```
 
+We also support GitHub App Installation based authentication for higher rate-limit quotas.
+If you have an installed GitHub App and key file, you can use these three environment variables, following the commands
```

Comment:
"you can instead use" ?

i think it is either GITHUB_AUTH_TOKEN or these three right ?